proto/relayer.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto3";
 
+import "proto/create-order.proto";
```

Comment:
We can discard these changes until we make modifications to the relayer proto.

These changes will get blown away on a local installation. See `https://github.com/sparkswap/broker/blob/master/scripts/build.sh#L182-L199`

Reply:
TBH probably the way to go about this is to modify that build script since we already check in the proto. What are your thoughts on how to handle this?

Reply:
I think for now we can just leave the proto files as they were since there isn't really that much duplication and the duplication we do have is somewhat legitimate.